config/action.d/firewallcmd-common.conf

```diff
@@ -5,72 +5,102 @@
 
 [Init]
 
+# Option:  bantime
+# Notes:   Specifies the bantime in seconds (handled internally rather than by fail2ban)
+# Values:  [ NUM ]  Default: 600
+
+bantime = 600
+
 # Option:  name
-# Notes    Default name of the chain
+# Notes:   Default name of the chain
 # Values:  STRING
+
 name = default
 
 # Option   port
-# Notes    Can also use port numbers separated by a comma and in rich-rules comma and/or space.
-# Value    STRING Default: 1:65535
+# Notes:   Can also use port numbers separated by a comma and in rich-rules comma and/or space.
+# Value    [ NUM | STRING ] Default: 1:65535
+
 port = 1:65535
 
 # Option:  protocol
-# Notes    [ tcp | udp | icmp | all ]
+# Notes:   [ tcp | udp | icmp | all ]
 # Values:  STRING Default: tcp
+
 protocol = tcp
 
 # Option:  family(ipv4)
-# Notes    specifies the socket address family type
-# Values:  STRING  
+# Notes:   Specifies the socket address family type
+# Values:  STRING Default: ipv4 
+
 family = ipv4
 
 # Option:  chain
-# Notes    specifies the firewalld chain to which the Fail2Ban rules should be
+# Notes:   Specifies the firewalld chain to which the Fail2Ban rules should be
```

Comment:
I have some time today so let me know what you'd like to see regarding format and I'll work on it.